src/content/en/2021/webassembly.md

```diff
@@ -85,7 +84,9 @@ The stark difference between the numbers of unique files and total responses alr
 
 Let's dive deeper and figure out what those reused libraries are. First, we've tried to deduplicate libraries by content hash alone, but it became quickly apparent that many of those left are still duplicates that differ only by library version.
 
-Instead, we decided to extract library names from URLs. While it's more problematic in theory due to potential name clashes, it turned out to be a more reliable option for top libraries in practice. We extracted filenames from URLs, removed extensions, minor versions and suffixes that looked like content hashes, sorted the results by number of repetitions and extracted the top 10 modules for each client. For those left, we did manual lookups to understand which libraries those modules are coming from.
+Then we decided to extract library names from URLs. While it's more problematic in theory due to potential name clashes, it turned out to be a more reliable option for top libraries in practice. We extracted filenames from URLs, removed extensions, minor versions and suffixes that looked like content hashes, sorted the results by number of repetitions and extracted the top 10 modules for each client. For those left, we did manual lookups to understand which libraries those modules are coming from.
+
+{# TODO - " minor versions and suffixes that looked like content hashes, " - does this mean they were removed, or something else? not clear #}
```

Comment:
@RReverser

Reply:
Yes, all of those are meant to be contextually under "removed [...list of things...]".

Reply:
Added an extra comma in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2605/commits/5129015399710c63060966d803fd930ece2d19e4. I think it's clear so removed the TODO.